packages/mui-material/src/Avatar/Avatar.test.js

```diff
@@ -41,7 +41,10 @@ describe('<Avatar />', () => {
       expect(avatar).not.to.have.class(classes.colorDefault);
       expect(img).to.have.class(classes.img);
       expect(img).to.have.attribute('alt', 'Hello World!');
-      expect(img).to.have.attribute('src', '/fake.png');
+      const expectedSrc = /jsdom/.test(window.navigator.userAgent)
+        ? '/fake.png'
+        : new URL('/fake.png', window.origin).toString();
+      expect(img).to.have.property('src', expectedSrc);
```

Comment:
Probably caused by https://github.com/facebook/react/pull/23316. Not clear to me if both values are interchangeable and if JSDOM is not correct here.

Reply:
Thanks for reporting, will fix this in React

Reply:
The fix will be included in the next RC and is available on main